code/game/gamemodes/storyevent.dm

```diff
@@ -71,7 +71,7 @@
 		if (report) report << SPAN_NOTICE("Failure: The event is disabled")
 		return FALSE
 
-	if ((ocurrences_max) > 0 && (ocurrences >= ocurrences_max))
+	if (ocurrences_max > 0 && ocurrences >= ocurrences_max)
```

Comment:
This couldn't have fixed anything. See BYOND F1 docs on operators: `>` and `>=` always have priority over `&&`, with or without `()`. Them having priority over `&&` also seems to be desired behavior here.

If you want to fix this issue, it would require further investigation.